### PR TITLE
don't mark multi-level action path as translatable

### PR DESCRIPTION
**Pull request by dterrahe**
This isn't supposed to work anyway; the translations are picked up from the same section names being marked for translation elsewhere (same as with "blend\`tools" and "blend\`shapes") and "blend" is just a marker string that gets mapped to the `<blending>` virtual module.

### CODE REVIEW

**Comment by jenshannoschwalm** [comment]
Something completely off-topic :-) I have no idea how all those keyboard / midi whatever actions work.

Is there any way to get noticed what module got any parameter changed without having an eye on history?
Or is there something like a "dispatcher" i could look into?

Would be interested to possibly rethink pipe-cache strategies. What we have is simply not working as good as possible (any more) ...

**Comment by dterrahe** [comment]
> Something completely off-topic :-)

Not sure whether your 3 questions are as unrelated to eachother as they seem...

For the 2nd one; in the end, a module gui has to indicate that a parameter has changed and needs to be updated in history by calling `dt_dev_add_history_item` (or `dt_dev_add_masks_history_item` for drawn mask changes). There is no notification mechanism you could link into that I'm aware of, so you'd have to intercept there. For bauhaus items, you _may_ (depending on whether they are introspection based) be able to intercept in the module's `gui_changed` method. But if you're thinking of changing low level caching, that doesn't seem like the right place.

Introspection and the action mechanism are separate although the latter does benefit/build on the uniformity enforced by the former.

> as good as possible

Might be worth discussing/bouncing ideas first before making invasive changes? Don't need to tell _you_ that trying to be smart in caching and adding complexity can place a massive burden on those who later try to understand what's going on in order to fix/further improve. :-)

Would rather not have this discussion in this really really really trivial PR fixing a simple oversight in #14421 though. It might give Pascal the idea that there's anything in here that requires even a first look before blindly pressing the merge button.

**Comment by dterrahe** [comment]
@TurboGit Upping priority in hope that other translators aren't going to waste time trying to figure out how to deal with the ` and provide a translation that will never be used.

**Comment by TurboGit** [comment]
> @TurboGit Upping priority in hope that other translators aren't going to waste time trying to figure out how to deal with the ` and provide a translation that will never be used.

Ok, merging now. But can you comment on why we translate some others (2nd argument of  `dt_action_define_iop`). Is that just because here the actual group is `<blending>` (this is how I understand your comment, but I want to be sure as this is not something I'm fully fluent :)

**Review by TurboGit — APPROVED** [review]
Thanks!

**Comment by dterrahe** [comment]
> But can you comment on why we translate some others (2nd argument of `dt_action_define_iop`). Is that just because here the actual group is `<blending>`

Yes. Remember, N_() just _marks_ text as translatable. The actual translating happens (once) here
https://github.com/darktable-org/darktable/blob/658f3c0a055a8a54e38591580138616b56c0327b/src/gui/accelerators.c#L4345
as long as a translation is available, which would be the case if a label has been marked for translation in at least _one_ place in the code. We have `N_("pickers")` in many places, so we are good.

"blend" (or "blend`) is a special marker, that redirects the actual action used to the <blending> virtual module. Here:
https://github.com/darktable-org/darktable/blob/658f3c0a055a8a54e38591580138616b56c0327b/src/gui/accelerators.c#LL4450C52-L4450C52

So it doesn't actually ever get translated. "\<blending\>" gets translated here, with a special context https://github.com/darktable-org/darktable/blob/658f3c0a055a8a54e38591580138616b56c0327b/src/control/control.c#L160

So should "blend" be marked for translation here?
https://github.com/darktable-org/darktable/blob/658f3c0a055a8a54e38591580138616b56c0327b/src/develop/blend_gui.c#L2726
Well, no, not really as the translation is not going to be used for this specific label (maybe it is elsewhere). But I thought it made more sense to mark it here for "consistency" anyway, because people tend to copy stuff all over the place and it is likely that where they'd paste it, a section _does_ need a translation.

**Pull request merged**
